src/client/Microsoft.Identity.Client/AuthScheme/Bearer/BearerAuthenticationOperation.cs

```diff
@@ -2,13 +2,15 @@
 // Licensed under the MIT License.
 
 using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
 using Microsoft.Identity.Client.Cache.Items;
 using Microsoft.Identity.Client.Internal;
 using Microsoft.Identity.Client.Utils;
 
 namespace Microsoft.Identity.Client.AuthScheme.Bearer
 {
-    internal class BearerAuthenticationOperation : IAuthenticationOperation
+    internal class BearerAuthenticationOperation : IAuthenticationOperation2
```

Comment:
Did this need to use the new interface?

What about `SSHCertAuthenticationOperation.cs` and `PopBrokerAuthenticationOperation.cs`